cmd/repository/default-pkg-registry.go

```diff
@@ -22,89 +19,23 @@ Further improvements could store registry as indexes, and laze load
 further information when necessary to reduce the startup time
 */
 type defaultRegistry struct {
```

Comment:
maybe name it in-memory-pkg-registry? The default-pkg-registry sounds the registry that will be used in the default-repository, but in fact it is the json-pkg-registry is used